src/test/resources/application.properties

```diff
@@ -1,5 +1,6 @@
-spring.datasource.url=jdbc:h2:mem:testdb; DB_CLOSE_DELAY=-1
+spring.datasource.url=jdbc:h2:mem:testdb;
 spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+spring.jpa.open-in-view=false
```

Comment:
that's quite an interesting thing, in case you want to know more:
https://stackoverflow.com/questions/30549489/what-is-this-spring-jpa-open-in-view-true-property-in-spring-boot